packages/replayio/src/utils/initialization/getLatestPackageVersion.ts

```diff
@@ -0,0 +1,31 @@
+import { exec } from "child_process";
+import { promisify } from "util";
+import { PackageManager } from "./types";
+
+const execAsync = promisify(exec);
+
+export async function getLatestPackageVersion(command: PackageManager, packageName: string) {
```

Comment:
I probably wouldn't even mind just using `npm` for this. Your version is better/more robust but even if a project uses an alternative package manager they will likely still have `npm` available.

But we could even go a step further - spawning an extra process for this is already a little bit wasteful. So we could optimize this by making a direct API call:
```ts
  const response = await fetch("https://registry.npmjs.org/replayio", {
    headers: {
      // request abbreviated metadata
      Accept: "application/vnd.npm.install-v1+json",
    },
  });
  const json = await response.json();
  const latestVersion = json["dist-tags"]?.latest;
```

This is documented [here](https://github.com/npm/registry/blob/1c794110badd54b9d9fb08e7489746b6089c6648/docs/responses/package-metadata.md)

Reply:
That seems like an improvement.

Reply:
https://github.com/replayio/replay-cli/pull/376